src/getting-started/tooling/espflash.md

```diff
@@ -1 +1,91 @@
-# espflash
+# `espflash`
+
+`espflash` is a serial flasher utility, based on [esptool.py][esptool], for Espressif SoCs and modules.
```

Comment:
```suggestion
`espflash` is a serial flasher utility, based loosely on [esptool.py][esptool], for Espressif SoCs and modules.
```
We could also maybe just remove the mention of `esptool` altogether, I don't think they share much in common other than the fact they're both flashing tools.